projects/angular/src/layout/breadcrumbs/breadcrumbs.html

```diff
@@ -0,0 +1,34 @@
+<div role="list" class="clr-breadcrumb-menu">
+  <ng-container *ngIf="items?.length">
```

Comment:
```suggestion
<div *ngIf="items?.length" role="list" class="clr-breadcrumb-menu">
  <ng-container>
```

These can be combined. No need to render an empty `div` if there are no items.